frontend/src/app/modules/root/root-reducer.js

```diff
@@ -14,13 +15,13 @@ const initialState = {
   users: [],
   activeAlert: null,
   activeRedirect: null,
-  theme: window.localStorage.getItem("theme") || "dark",
+  theme: window.localStorage.getItem('theme') || 'dark',
```

Comment:
`ThemeTypes.Dark` can be used instead of the string. Also maybe extract the `'theme'` item name into constant as it's used multiple times?

Reply:
I disagree with the necessity for the `'theme'` string constant. There are 2 usages of the string and I think it would decrease readability more than it would increase reusability. Not to mention it would require to name the constant which is pretty time consuming :( What do you think @petrk-salsita ?

Reply:
[done](https://github.com/salsita/foosball-rating/pull/81/commits/6233538904b5337673c199255c73c62943d1415e#diff-a40f089e71d55fac58c940a9aa91b1d0)

Reply:
added global constants [here](https://github.com/salsita/foosball-rating/pull/81/commits/44f0448e166dac047262d9a907d4cc973e9a7619#diff-fc024cf3f843472eca76dc473839b5c8)